init.lua

```diff
@@ -878,6 +878,7 @@ require('lazy').setup({
   -- require 'kickstart.plugins.lint',
   -- require 'kickstart.plugins.autopairs',
   -- require 'kickstart.plugins.neo-tree',
+  require 'kickstart.plugins.oil',
```

Comment:
```suggestion
  -- require 'kickstart.plugins.oil',
```
I think this should follow the other plugins in being disabled by default

Reply:
Great suggestion!